### ISSUE TITLE
scss compiles error

### ISSUE THREAD

**Issue by Mad-hu** [electron-react-boilerplate/electron-react-boilerplate#2523]
![image](https://user-images.githubusercontent.com/13977755/87665972-644ed380-c79a-11ea-853a-24bbca37a398.png)
recurrent trouble
git clone,and change files 
Counter.css->Counter.scss
Counter.tsx ->
```js
import React from 'react';
import { useSelector, useDispatch } from 'react-redux';
// import { Link } from 'react-router-dom';
import './Counter.scss';
// import routes from '../../constants/routes.json';
import {
  // increment,
  // decrement,
  // incrementIfOdd,
  // incrementAsync,
  selectCount,
} from './counterSlice';

export default function Counter() {
  // const dispatch = useDispatch();
  // const value = useSelector(selectCount);
  return (
    <div>

    </div>
  );
}
```
yarn dev ,and error
- [ 1.19.1 ] Using yarn
- [ 12.14.1 ] Using node 10.x

"name": "css-loader",
"version": "3.6.0",
![image](https://user-images.githubusercontent.com/13977755/87666690-a7f60d00-c79b-11ea-992b-29ca79e7a5d3.png)

**Comment by fondoger** [comment]
Same for me

**Comment by sepehr09** [comment]
i have same problem
i do some search and find that `typings-for-css-modules-loader` is deprecated and not maintain any more. i think that's cause this problem.

any solution?

**Comment by jooohhn** [comment]
Working on a fix at the moment, bug seemed to be introduced by dependency upgrades in v1.2.0

Temporary solution would be to use v1.1.0 (#2409)

**Comment by jooohhn** [comment]
Should be fixed and incorporated by next project release. If you need it ASAP check out changes merged into the `next` branch (#2540)
